tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +63,65 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownValidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    List<String> groups = CommonUtils.getGroups(newOwner);
```

Comment:
what groups is "alice" going to be in? we should make sure to test the case where the owner name is different from the group name to catch bugs where they get mixed up.